reference/spotlight.md

```diff
@@ -4,17 +4,17 @@ Derived from [Light](light.md).
 
 ```
 SpotLight {
-  SFFloat ambientIntensity 0        # [0,1]
-  SFVec3f attenuation      1 0 0    # [0,inf)
-  SFFloat beamWidth        1.570796 # [0,pi/2)
-  SFColor color            1 1 1    # [0,1]
-  SFFloat cutOffAngle      0.785398 # [0,pi/2)
-  SFVec3f direction        0 0 -1   # (-inf,inf)
-  SFFloat intensity        1        # [0,1]
-  SFVec3f location         0 0 10   # (-inf,inf)
-  SFBool  on               TRUE
-  SFFloat radius           100      # [0,inf)
-  SFBool  castShadows      FALSE
+  SFFloat ambientIntensity 0          # [0, 1]
+  SFVec3f attenuation      1 0 0      # [0, inf)
+  SFFloat beamWidth        1.570796   # [0, pi/2)
+  SFColor color            1 1 1      # [0, 1]
+  SFFloat cutOffAngle      0.785398   # [0, pi/2)
+  SFVec3f direction        0 0 -1     # (-inf, inf)
+  SFFloat intensity        1          # [0, 1]
+  SFVec3f location         0 0 10     # (-inf, inf)
```

Comment:
We have a problem with `SFVec3f`. Either we write "any vector" or something like "(-inf, inf) (-inf, inf) (-inf, inf)" or simply nothing. Similar problem with `SFVec2f` and `SFRotation`...